tests/org.dataflowanalysis.analysis.tests/src/org/dataflowanalysis/analysis/tests/converter/WebEditorTest.java

```diff
@@ -0,0 +1,72 @@
+package org.dataflowanalysis.analysis.tests.converter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.dataflowanalysis.analysis.converter.DataFlowDiagramAndDictionary;
+import org.dataflowanalysis.analysis.converter.DataFlowDiagramConverter;
+import org.dataflowanalysis.analysis.converter.webdfd.WebEditorDfd;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class WebEditorTest extends ConverterTest {
+    private DataFlowDiagramConverter converter;
+
+    private final String MINIMAL = Paths.get(packagePath, "minimal.json").toString();
```

Comment:
Minimal what? The local variables should be named more precise. Also, all caps lock is reserved for constants which are directly assigned, e.g., without calling methods